src/main/features/dashboard/views/macro/claimStatus/claimFullyAdmitted.njk

```diff
@@ -0,0 +1,111 @@
+    <p> {{ t('If they don’t receive the money by then, they can request a County Court Judgment against you.') }} </p>
+    <p> {{ t('Contact {{ claimantName }} if you need their payment details.',
+        { claimantName: claimantName }) }} </p>
+    <p> {{ t('Make sure you get receipts for any payments.') }} </p>
+
+  {% elseif claim.response.paymentIntention.paymentOption === domain.PaymentOption.BY_SPECIFIED_DATE %}
+    <p> {{ t('You’ve offered to pay {{ claimantName }} {{ amount }} on {{ paymentDate }}.',
+        { claimantName: claimantName, amount: amount, paymentDate: paymentDate }) }} </p>
+    <p>
+      {{ t('We’ll contact you when they respond.') }}
+    </p>
+  {% elseif  claim.response.paymentIntention.paymentOption === domain.PaymentOption.INSTALMENTS %}
+    {% set repaymentPlan = claim.response.paymentIntention.repaymentPlan %}
+    {% set repaymentDate = claim.response.paymentIntention.repaymentPlan.firstPaymentDate |date %}
+    {% set paymentSchedule = repaymentPlan.paymentSchedule | renderPaymentSchedule %}
+
+    <p> {{ t('You’ve offered to pay {{ claimantName }} {{ amount }} {{ paymentSchedule }} starting {{ repaymentDate }}.',
+        { claimantName: claimantName, amount: amount, paymentSchedule: paymentSchedule, repaymentDate: repaymentDate }) }} </p>
+    <p>
+      {{ t('We’ll contact you when they respond.') }}
+    </p>
+  {% endif %}
+  <p>
+    {{ _downloadResponseLink(claim) }}
+  </p>
+{% endmacro %}
+
+{% macro claimFullAdmissionForClaimantClaimDetails(claim) %}
+  <h2 class="heading-small">{{ t('The defendant’s response') }}</h2>
+
+  {% set claimantName = claim.claimData.claimant.name %}
+  {% set defendantName = claim.claimData.defendant.name %}
+  {% set amount = claim.totalAmountTillToday | numeral %}
+
+  {% if claim.response.paymentIntention.paymentDate %}
+    {% set paymentDate = claim.response.paymentIntention.paymentDate | date %}
+  {% endif %}
+
+  {% if claim.response.paymentIntention.paymentOption === domain.PaymentOption.IMMEDIATELY %}
+    <h2 class="heading-small">{{ t('The defendant will pay you immediately') }}</h2>
+    <p> {{ t('The defendant said they’ll pay you immediately. They must make sure you have the money by 4pm on {{ paymentDate }}.',
+        { paymentDate: paymentDate }) }} </p>
+    <h2 class="heading-small"> {{ t('If the defendant pays you.') }} </h2>
+    <p> {{ t('You need to email moneyclaims@hmcts.gsi.gov.uk to let us know. Include the date you were paid.') }} </p>
+    {% if claim.response.paymentIntention.pastDefendantPayImmediatelyDate %}
+      <h2 class="heading-small">{{ t('What to do if they haven’t paid') }}</h2>
+      <p> {{ t('If the defendant doesn’t pay you, you can request a County Court Judgment.') }} </p>
+    {% else %}
+      <h2 class="heading-small">{{ t('What to do if they don’t pay') }}</h2>
+      <div> {{ t('If the defendant has not paid you by {{ paymentDate }}, you can request a County Court Judgment.', {paymentDate: paymentDate}) }} </div>
```

Comment:
Use `p` instead of `div`.

Also it looks to me like this line should be in `if claim.response.paymentIntention.pastDefendantPayImmediatelyDate`. Worth double checking content.